plugins/support-helphub/inc/table-of-contents-lite/assets/css/style.css

```diff
@@ -8,22 +8,30 @@
 	padding: 0;
 }
 
-h2.toc-heading {
-	font-size: 26px;
-	line-height:39px;
+.entry-content .toc-heading,
```

Comment:
@zzap Aren't we moving the ToC to the side bar?

I'm thinking the .entry-content won't actually be useful once we do that.

Reply:
No, TOC stays where it is. Sidebar is for links t other articles, archives and language switcher. 

https://wordpress.slack.com/archives/C02RP4WU5/p1525706110000334

>kenshino [5:15 PM]
>@mapk besides languages what other use could there be for the side bar?
>
>mapk [5:15 PM]
>subnav
>
>kenshino [5:15 PM]
>What kinda subnav?
>
>mapk [5:15 PM]
>to other articles within the same section. Or a jump to other sections?
>
>kenshino [5:15 PM]
>So the TOC?
>
>mapk [5:16 PM]
>No… that’s separate.
>If I’m reading an article in the “Installing WordPress” section, maybe the sidebar shows the other sections like “Getting Started”, “Basic Usage”, etc. (edited)
>But that would mean that either that sidebar nav would open accordion-style to show the articles under it, or there would be a landingpage for each section.

Reply:
Okay makes sense if we're thinking mobile and we don't want the ToC to be dropped all the way to the bottom